adafruit_max1704x.py

```diff
@@ -122,91 +127,91 @@ def reset(self):
         self.reset_alert = False  # clean up RI alert
 
     @property
-    def cell_voltage(self):
+    def cell_voltage(self) -> float:
         """The state of charge of the battery, in volts"""
         return self._cell_voltage * 78.125 / 1_000_000
 
     @property
-    def cell_percent(self):
+    def cell_percent(self) -> float:
         """The state of charge of the battery, in percentage of 'fullness'"""
         return self._cell_SOC / 256.0
 
     @property
-    def charge_rate(self):
+    def charge_rate(self) -> float:
         """Charge or discharge rate of the battery in percent/hour"""
         return self._cell_crate * 0.208
 
     @property
-    def reset_voltage(self):
+    def reset_voltage(self) -> float:
         """The voltage that will determine whether the chip will consider it a reset/swap"""
         return self._reset_voltage * 0.04  # 40mV / LSB
 
     @reset_voltage.setter
-    def reset_voltage(self, reset_v):
+    def reset_voltage(self, reset_v: float) -> None:
         if not 0 <= reset_v <= (127 * 0.04):
             raise ValueError("Reset voltage must be between 0 and 5.1 Volts")
         self._reset_voltage = int(reset_v / 0.04)  # 40mV / LSB
 
     @property
-    def voltage_alert_min(self):
+    def voltage_alert_min(self) -> float:
         """The lower-limit voltage for the voltage alert"""
         return self._valrt_min * 0.02  # 20mV / LSB
 
     @voltage_alert_min.setter
-    def voltage_alert_min(self, minvoltage):
+    def voltage_alert_min(self, minvoltage: float) -> None:
         if not 0 <= minvoltage <= (255 * 0.02):
             raise ValueError("Alert voltage must be between 0 and 5.1 Volts")
         self._valrt_min = int(minvoltage / 0.02)  # 20mV / LSB
 
     @property
-    def voltage_alert_max(self):
+    def voltage_alert_max(self) -> float:
         """The upper-limit voltage for the voltage alert"""
         return self._valrt_max * 0.02  # 20mV / LSB
 
     @voltage_alert_max.setter
-    def voltage_alert_max(self, maxvoltage):
+    def voltage_alert_max(self, maxvoltage: float) -> None:
         if not 0 <= maxvoltage <= (255 * 0.02):
             raise ValueError("Alert voltage must be between 0 and 5.1 Volts")
         self._valrt_max = int(maxvoltage / 0.02)  # 20mV / LSB
 
     @property
-    def active_alert(self):
+    def active_alert(self) -> bool:
         """Whether there is an active alert to be checked"""
         return self._alert_status
 
     @property
-    def alert_reason(self):
+    def alert_reason(self) -> bool:
```

Comment:
Looks like this should be an `int` since it returns a number representing all the bits that define the reason for an alert.